Formula/asdf.rb

```diff
@@ -0,0 +1,32 @@
+class Asdf < Formula
+  desc "Extendable version manager with support for Ruby, Node.js, Erlang & more"
+  homepage "https://github.com/asdf-vm"
+  url "https://github.com/asdf-vm/asdf/archive/v0.3.0.tar.gz"
+  sha256 "1d986789bfcbef76dc23cc968ca192a03862a9496df65e7cd5d6f8117bc85789"
+
+  bottle :unneeded
+
+  depends_on "coreutils" => :run
```

Comment:
Hello again @dunn,

As you saw, asdf is a version manager which uses some plugins to control the version of the software you want, that version of the software and those plugins requires a build process. I am only following their recommendation, in the [README](https://github.com/asdf-vm/asdf/blob/master/README.md) you can read:

```
For most plugins, it is good if you have installed the following packages OR their equivalent on your OS

OS X: Install these via homebrew coreutils automake autoconf openssl libyaml readline libxslt libtool unixodbc
```

coreutils, automake,  libtool.... are coming from there 😅 . Removing them you will break the main feature imo.... I can not tell something else... what is your recommendation?